### DISCUSSION TITLE
Duplicated lang route

### DISCUSSION THREAD

**Discussion by Gigiz** [vinissimus/next-translate#490] [Q&A]
Hi, I started with the officiale Next.js example `with-next-translate` and I noticed some strange behavior.

I try to describe it

```
http:// localhost:3000/en -> english index page
http:// localhost:3000/en/en  -> english index page
http:// localhost:3000/en/ca  -> catalan translated index page
```

`next build` seems to create a correct folder structure

```
> tree
.
├── _app.js
├── _document.js
├── _error.js
├── ca
│   └── 404.html
├── ca.html
├── ca.json
├── en
│   └── 404.html
├── en.html
├── en.json
└── index.js
```

I would like to understand why the two routes with double locale do not respond with 404 page

**Comment by vadymshymko** [comment]
Is there any update about this?

**Reply by aralroca** [reply]
I imagine that instead of `next build` is doing `next export`. `next build` works fine, but `next export` have some caveats yet as the documentation of Next.js says [here](https://nextjs.org/docs/advanced-features/i18n-routing#how-does-this-work-with-static-generation). I imagine we need to wait until they implement the compatibility with i18n routing.
